### DISCUSSION TITLE
Finalizing Web Frontend

### DISCUSSION THREAD

**Discussion by dbaslan** [SWE574-Fall2023-Group1/SWE574-Fall2023-G1#245] [Ideas]
@SWE574-Fall2023-Group1/all Alright everyone, I'll be handling the final touches for the frontend. Please write down anything you'd like me to do here. Bugs, new features, things to be removed, etc. Feel free to open individual issues, assign them to me and mention them here if you prefer.

**Comment by sadikkuzu** [comment]
Thanks @dbaslan 
Here is my recommendation: https://github.com/SWE574-Fall2023-Group1/SWE574-Fall2023-G1/discussions/236

**Comment by sadikkuzu** [comment]
@dbaslan hi,
The professor suggested we change the background gif image.
Can you please address this suggestion as well?
It would be great if you could do this along with the font changes.

**Comment by sadikkuzu** [comment]
Is it possible to put some space above the logo?

![welcome-page](https://github.com/SWE574-Fall2023-Group1/SWE574-Fall2023-G1/assets/23168063/afd471ed-1a9d-4db1-80db-7239ebe07631)
